grapher/stackedCharts/MarimekkoChart.tsx

```diff
@@ -0,0 +1,1600 @@
+import * as React from "react"
+import {
+    min,
+    max,
+    maxBy,
+    last,
+    flatten,
+    excludeUndefined,
+    sortBy,
+    sumBy,
+    sum,
+} from "../../clientUtils/Util"
+import { action, computed, observable } from "mobx"
+import { observer } from "mobx-react"
+import { Bounds, DEFAULT_BOUNDS } from "../../clientUtils/Bounds"
+import {
+    BASE_FONT_SIZE,
+    EntitySelectionMode,
+    SeriesName,
+} from "../core/GrapherConstants"
+import { DualAxisComponent } from "../axis/AxisViews"
+import { NoDataModal } from "../noDataModal/NoDataModal"
+import { AxisConfig } from "../axis/AxisConfig"
+import { ChartInterface } from "../chart/ChartInterface"
+import { OwidTable } from "../../coreTable/OwidTable"
+import { autoDetectYColumnSlugs, makeSelectionArray } from "../chart/ChartUtils"
+import { stackSeries } from "./StackedUtils"
+import { ChartManager } from "../chart/ChartManager"
+import { Color, Time } from "../../clientUtils/owidTypes"
+import { StackedPoint, StackedSeries } from "./StackedConstants"
+import { ColorSchemes } from "../color/ColorSchemes"
+import {
+    EntityId,
+    EntityName,
+    LegacyOwidRow,
+    OwidTableSlugs,
+} from "../../coreTable/OwidTableConstants"
+import {
+    LegendAlign,
+    HorizontalCategoricalColorLegend,
+    HorizontalColorLegendManager,
+} from "../horizontalColorLegend/HorizontalColorLegends"
+import { CategoricalBin } from "../color/ColorScaleBin"
+import { CoreColumn } from "../../coreTable/CoreTableColumns"
+import { TippyIfInteractive } from "../chart/Tippy"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons/faInfoCircle"
+import { DualAxis, HorizontalAxis, VerticalAxis } from "../axis/Axis"
+import { ColorScale, ColorScaleManager } from "../color/ColorScale"
+import {
+    ColorScaleConfig,
+    ColorScaleConfigDefaults,
+} from "../color/ColorScaleConfig"
+import { ColorSchemeName } from "../color/ColorConstants"
+import { color } from "d3-color"
+import { SelectionArray } from "../selection/SelectionArray"
+import { ColorScheme } from "../color/ColorScheme"
+import _ from "lodash"
```

Comment:
One more thing here: Never import lodash itself, always import the functions you need through `../../clientUtils/Util` (and expose them there if they're not already exported there).
It's not great but makes our bundle size a noticeable amount smaller. This should also make you pass the bundlewatch check.

Reply:
Okay, just fixed that for you in a tiny commit.